Doc/library/string.rst

```diff
@@ -517,7 +520,10 @@ The available integer presentation types are:
    +---------+----------------------------------------------------------+
    | ``'n'`` | Number. This is the same as ``'d'``, except that it uses |
    |         | the current locale setting to insert the appropriate     |
-   |         | digit group separators.                                  |
+   |         | digit group separators. Note that the default locale is  |
+   |         | not the system locale, and therefore, you must set, at a |
+   |         | minimum, the ``LC_NUMERIC`` category (see the            |
```

Comment:
This isn't true, is it? I'm referring to the wording "must set": shouldn't it be "can set".

Reply:
On my machine, nothing happens if I use `n`, without setting `LC_NUMERIC`, so it is "must set" for me. I'm happy to adjust the wording if there are cases where it is set, but I haven't seen it.

FWIW, I'm seeing that the C locale seems to be very minimal and [doesn't have a thousands separator](https://devblogs.microsoft.com/oldnewthing/20250206-00/?p=110846#:~:text=In%20the%20%E2%80%9CC%E2%80%9D%20locale%2C,there%20is%20no%20thousands%20separator.). Since that is what the default is, instead of the system locale, the thousands separator won't be set until the user sets `LC_NUMERIC`, meaning it is "must set" for everyone. I'm new to the C locale as well though, so let me know if that isn't the case.